id,module,name,file,doc,markers,status,message,duration
functional_correctness_test_folder/csn/gpt-5/full_docstr/test_snippet_36.py::test_case_0,functional_correctness_test_folder.csn.gpt-5.full_docstr.test_snippet_36,test_case_0,functional_correctness_test_folder/csn/gpt-5/full_docstr/test_snippet_36.py,,,failed,TypeError: Sentence.__init__() takes from 3 to 4 positional arguments but 6 were given,0.00010020899935625494
functional_correctness_test_folder/csn/gpt-5/full_docstr/test_snippet_36.py::test_case_1,functional_correctness_test_folder.csn.gpt-5.full_docstr.test_snippet_36,test_case_1,functional_correctness_test_folder/csn/gpt-5/full_docstr/test_snippet_36.py,,xfail,xfailed,,8.225000055972487e-05
functional_correctness_test_folder/csn/gpt-5/full_docstr/test_snippet_36.py::test_case_2,functional_correctness_test_folder.csn.gpt-5.full_docstr.test_snippet_36,test_case_2,functional_correctness_test_folder/csn/gpt-5/full_docstr/test_snippet_36.py,,xfail,xfailed,,9.299999510403723e-05
